pontoon/base/middleware.py

```diff
@@ -47,3 +50,20 @@ def process_request(self, request):
             return HttpResponseForbidden('<h1>Forbidden</h1>')
 
         return None
+
+
+class AutomaticLoginUserMiddleware(object):
+    """
+    This middleware automatically logs in the user with username
+    'ADMIN_EMAIL' and password 'ADMIN_PASSWORD' if AUTO_LOGIN enabled.
+    """
+    def process_request(self, request):
+        if settings.AUTO_LOGIN and not request.user.is_authenticated():
+            user = auth.authenticate(
+                username=os.environ.get('ADMIN_EMAIL'),
+                password=os.environ.get('ADMIN_PASSWORD')
```

Comment:
A kinda low-priority note: Auth backends could help you to avoid keeping plain-text password in the environment. http://stackoverflow.com/questions/2787650/manually-logging-in-a-user-without-password